cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
```

Comment:
@Scotchester but if we special-case prod, then we get the welcome message that this isn't production everywhere *except* prod, and we get the helpful documentation links below everywhere *except* prod.

This would prevent us from having to special-case other hostnames in the future if we standup development servers that are not called "build".